client/src/entry.tsx

```diff
@@ -100,6 +101,7 @@ export function mount({
 
   const apolloClient = new ApolloClient({
     link: ApolloLink.from([
+      new DebounceLink(0), // order is important
```

Comment:
why 0?

Reply:
because this is the default debounce timeout, the queries that want a debounce will set it, eg. `gqlListPinboards` in `SelectPinboard` - may be worth adding as a comment, or putting the 0 in a named variable?

Reply:
good point, thanks - done